specifyweb/frontend/js_src/lib/components/WorkBench/Disambiguation.tsx

```diff
@@ -16,18 +16,20 @@ import { hasTablePermission } from '../Permissions/helpers';
 
 export function DisambiguationDialog({
   matches,
+  defaultResource,
   onSelected: handleSelected,
   onSelectedAll: handleSelectedAll,
   onClose: handleClose,
 }: {
   readonly matches: RA<SpecifyResource<AnySchema>>;
+  readonly defaultResource?: SpecifyResource<AnySchema>;
   readonly onSelected: (resource: SpecifyResource<AnySchema>) => void;
   readonly onSelectedAll: (resource: SpecifyResource<AnySchema>) => void;
   readonly onClose: () => void;
 }): JSX.Element {
   const [selected, setSelected] = React.useState<
     SpecifyResource<AnySchema> | undefined
-  >(undefined);
+  >(defaultResource);
```

Comment:
@grantfitzsimmons would it be better UX if disambiguation dialog in the workbench had first item selected by default when you open it?
if so, @realVinayak can you fallback to first array item when defautlResource is not provided?

Reply:
I don't think so

Reply:
yeah, the first item can be pretty arbitrary